docs/examples/auto-instrumentation/README.rst

```diff
@@ -233,5 +233,5 @@ Additional resources
 ~~~~~~~~~~~~~~~~~~~~
 
 In order to send telemetry to an OpenTelemetry Collector without doing any
-additional configuration, read about the `OpenTelemetry Distro <../distro/README.html>`_
+additional configuration, read about the `OpenTelemetry Distro <../distro>`_
```

Comment:
Thanks for submitting this change! 🙂

I noticed that this link is currently working on [the live build](https://opentelemetry-python.readthedocs.io/en/latest/examples/auto-instrumentation/README.html#additional-resources)

Clicking on this link takes you the expected place:

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/23139455/133659448-60d2a196-9cba-47b4-ad05-577ae86dafc5.png">

Did it not work when you tried it?

Reply:
@NathanielRN if you follow the link in the description of this PR it leads you to the broken link at the bottom of the page.

Reply:
@NathanielRN please resolve this conversation if it makes sense so that PR is merged (automatically :D )

Reply:
This doesn't work on my local build because it just send me to the directory:

![image](https://user-images.githubusercontent.com/23139455/134238594-92532b47-19e3-4e84-a88b-13a37b564c6b.png)


The only example in our current code that I could find that is close to this one is here:

https://github.com/open-telemetry/opentelemetry-python/blob/eacff3e84b9452357b7a163fc937b491c089c63f/docs/examples/opentracing/README.rst#L5

which works on the live build + my local build but doesn't work on GitHub? It also doesn't have the `_` at the end of the link...

![image](https://user-images.githubusercontent.com/23139455/134238518-2cd79429-121f-4b4c-a278-b252f70f7a47.png)

But I'm pretty unfamiliar with the sphinx build still, so let's go ahead and try this 😄 But we should do a fast follow up to make sure it worked!